zerver/management/commands/makemessages.py

```diff
@@ -50,7 +50,7 @@
 strip_whitespace_left = re.compile(u"\\s+(%s-\\s*(endtrans|pluralize).*?-?%s)" % (
                                    BLOCK_TAG_START, BLOCK_TAG_END), re.U)
 
-regexes = ['{{#tr .*?}}(.*?){{/tr}}',
+regexes = ['{{#tr .*?}}([\s\S]*?){{/tr}}',
```

Comment:
This is because we don't compile the regexes with the `re.DOTALL` flag. Hence, the dot operator doesn't include whitespaces.

The change is required if we want to avoid modifying the other regexe's behavior.